clang/docs/ReleaseNotes.rst

```diff
@@ -64,6 +64,9 @@ sections with improvements to Clang's support for those languages.
 
 C++ Language Changes
 --------------------
+- Allow single element access of vector object to be constant expression.
```

Comment:
This is potentially confusing to refer to "vector object". Be more specific this is an ext_vector_type vector?

Reply:
done